Launcher/config.json

```diff
@@ -164,6 +164,13 @@
   // To get your access token go to https://polygon.io
   "polygon-api-key": "",
 
+  // zerodha configuration goto https://kite.trade
+  "zerodha-access-token": "",
+  "zerodha-api-key": "",
+  "zerodha-product-type": "MIS", //MIS(Intraday) or CNC(Delivery) or NRML(Carry Forward)
+  "zerodha-trading-segment": "EQUITY", // EQUITY(NSE,BSE) or COMMODITY (MCX)
+  "zerodha-history-subscription": false, // Enable this if History API Subscription available
```

Comment:
Not sure this config is required, it's kind of implicit by choosing the history provider 🤔

Reply:
Zerodha charges users for historical data API separately.